src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileMetadataFields.scala

```diff
@@ -32,7 +34,10 @@ object FileMetadataFields {
 
   case class DeleteFileMetadata(fileIds: Seq[UUID], filePropertyNames: Seq[String])
 
-  case class DeleteFileMetadataInput(fileIds: Seq[UUID])
+  case class DeleteFileMetadataInput(
```

Comment:
Is adding the additional field `propertyNames` not going to break any client using this input?

Reply:
Hmm, it shouldn't due to the default argument ` = Seq(ClosureType)`. I added [a card](https://national-archives.atlassian.net/browse/TDR-2723) to then add the argument explicitly

Reply:
OK.